tests/spec/test-presence-api-service.js

```diff
@@ -0,0 +1,90 @@
+(function () {
+    'use strict';
+
+    describe('Presence API Service', function () {
+        var server;
+        var cookieDummy = {
+            get: function () {
+                return '';
+            },
+            set: function (newCookie) {}
+        };
+        before(function () {
+            server = sinon.fakeServer.create();
+            server.respondWith('POST', /(.*)\/game/, function (xhr, id) {
+                xhr.respond(201, { 'Content-Type': 'application/json' }, JSON.stringify({ }));
+            });
+
+            server.respondImmediately = true;
+        });
+
+        after(function () {
+            server.restore();
+        });
+
+        function createPresenceAdapter(options) {
+            return new F.service.Presence(options);
+        }
+
+        describe('markOnline', function () {
+            it('should call POST on the Presence API with the userId parameter and the token', function () {
+                createPresenceAdapter({ token: '123' })
+                    .markOnline('abc123');
+
+                var req = server.requests.pop();
+                req.method.toUpperCase().should.equal('POST');
+                (/\/presence\//).test(req.url).should.be.true;
+                (/abc123/).test(req.url).should.be.true;
+                (/Bearer 123/).test(req.requestHeaders.Authorization).should.be.true;
```

Comment:
this test is covering too much -- i think checking url and method is fine, but move auth check to a different test